api/src/pcapi/core/offers/repository.py

```diff
@@ -575,6 +575,22 @@ def check_stock_consistency() -> list[int]:
     ]
 
 
+def get_bookings_count_by_offer(offer_id: int) -> int | None:
+    return (
+        bookings_models.Booking.query.join(models.Stock)
+        .with_entities(
+            sa.func.coalesce(
+                sa.func.sum(bookings_models.Booking.quantity).filter(
+                    models.Stock.offerId == offer_id,
+                    bookings_models.Booking.status != bookings_models.BookingStatus.CANCELLED,
+                ),
+                None,
+            )
+        )
+        .scalar()
+    )
```

Comment:
Je pense que tu peux faire

```suggestion
    return (
        Stock.query.filter(Stock.offerId == offer_id).with_entities(
            sqla.func.coalesce(sqla.func.sum(Stock.dnBookedQuantity), None)
        ).scalar()
)
```

`Booking.quantity` a été dénormalisé pour justement ne pas avoir à compter les bookings